app/Http/Controllers/ReservationController.php

```diff
@@ -84,11 +91,14 @@ private function handleReservationRoom(Request $request, $reservation) {
         $room = Room::getRoomData($request->room);
         $reservation->rooms()->sync($room);
     }
-
+    
     public function viewReservationInfo(int $id) {
         $reservation = Reservation::getReservationData($id);
 
-        return view('reservation.info', ['reservation' => $reservation]);
+        $availableRooms = $this->getAvailableRoomsDuringReservation($reservation);
+
+        return view('reservation.info', ['reservation' => $reservation, 'availableRooms' => $availableRooms,
```

Comment:
Je hoeft available rooms & current rooms hier niet meer op te halen & mee te geven, je kan in de blade file ook gwn $reservation->rooms gebruiken

Reply:
Terechte opmerking, ik heb de functie opgeschoond